aws/resource_aws_elasticache_replication_group.go

```diff
@@ -518,7 +528,18 @@ func resourceAwsElasticacheReplicationGroupRead(d *schema.ResourceData, meta int
 		c := res.CacheClusters[0]
 		d.Set("node_type", c.CacheNodeType)
 		d.Set("engine", c.Engine)
-		d.Set("engine_version", c.EngineVersion)
+
+		engineVersion, err := gversion.NewVersion(aws.StringValue(c.EngineVersion))
```

Comment:
This looks the same as in `aws/resource_aws_elasticache_cluster.go`. Can it be pulled out to be a shared function between the two?

Reply:
Good thinking